styles.css

```diff
@@ -46,12 +51,18 @@ body {
 
 /*  ---- start exercise 3 ----  */
 .card-3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
   border: 1px solid #4d7298;
   border-radius: 10px;
   height: 150px;
   width: 150px;
   padding: 20px;
 }
+.child-3 {
+  margin-top: auto;
+}
```

Comment:
Another way of doing this with flexbox is using the flex-grow property like this:

```
.child-2 {
  flex-grow: 1;
}
```